src/components/axes/axis.ts

```diff
@@ -54,12 +55,13 @@ module Plottable {
 
       this.formatter(formatter);
 
-      this._scale.broadcaster.registerListener(this, () => this._rescale());
+      this._rescaleCallback = (scale) => this._rescale();
```

Comment:
I don't think the scale argument is really that useful here (as for other similar code)